Monero/0.17.1.8/linuxamd64.Dockerfile

```diff
@@ -0,0 +1,36 @@
+# runtime stage
+FROM debian:stretch-slim
+ENV FILE=monero-linux-x64-v0.17.1.8.tar.bz2
+ENV FILE_CHECKSUM=98ce0d22db0d1112114bbad4c9773d1490d30e5c643423c2e5bffc19553207f9
```

Comment:
Checksum is unchanged from .17.1.7

I beleive it should be 
b566652c5281970c6137c27dd15002fe6d4c9230bc37d81545b2f36c16e7d476

Once this line is changed, it builds without issue.